docs/reference/objects/modifier_selection_session.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Modifier Selection Session
+parent: Objects
+---
+
+This drop is available on the post checkout selection page as "item".
+It combines the data for a particular line item with the guest information for
+the user currently logged in.
+
+# modifier_selection_session.id
```

Comment:
if it's available as `item` should we update these to be `item.x`?

See what you meant about this being confusing